test/support/api/forms.js

```diff
@@ -7,21 +7,19 @@ const fxSampleForms = _.rest(fxTestForms, 1);
 
 const TYPE = 'forms';
 
-export function getForm(data) {
-  const resource = getResource(_.sample(fxSampleForms), TYPE);
+// Exporting only form needed for testing variance
+export const testForm = getResource(_.extend(fxTestForms[0], {
+  name: 'Test Form',
+}), TYPE);
 
-  return mergeJsonApi(resource, data);
+export function getForm(data) {
+  return mergeJsonApi(testForm, data);
```

Comment:
This breaks this test: https://cloud.cypress.io/projects/ep9zr6/runs/6773/test-results?utm_source=github&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D

In that test an array of forms is created like `[testForm, getForm(), getForm()]`. And each form then has the same `id`. Which means only one shows up in the form dropdown.